lib/util/exec/containerbase.ts

```diff
@@ -91,6 +103,11 @@ const allToolConfig: Record<string, ToolConfig> = {
     packageName: 'lerna',
     versioning: npmVersioningId,
   },
+  maven: {
+    datasource: 'maven',
+    packageName: 'org.apache.maven:maven',
```

Comment:
This is copied from https://github.com/containerbase/base/blob/ee496c11fa12e5e1b810544397071c9b8f906b4d/test/java/Dockerfile#L157-L158

Reply:
we don't yet install maven. we only support the maven wrapper. so do we really need it now?

Reply:
I'm reusing this for containerbase/runinstall. Maybe the list should be broken out into containerbase later instead?

Reply:
ok, 🤔 so publish as new npm package to consume by renovate? or just provide as JSON file inside containerbase tools?